.eslintrc.js

```diff
@@ -1,34 +1,77 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: 'tsconfig.json',
-    sourceType: 'module',
-  },
-  plugins: ['@typescript-eslint/eslint-plugin'],
-  extends: [
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
-    'prettier',
-  ],
   root: true,
   env: {
     node: true,
-    jest: true,
+    es2021: true,
   },
-  ignorePatterns: ['.eslintrc.js'],
-  rules: {
-    '@typescript-eslint/interface-name-prefix': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
-    'prettier/prettier': [
-      'error',
-      {
-        'singleQuote': true,
-        'semi': true,
-        'trailingComma': 'all'
+
+  parserOptions: { sourceType: 'module' },
+
+  overrides: [
+    {
+      files: ['.eslintrc.js'],
+      extends: ['plugin:prettier/recommended'],
+      plugins: ['prettier'],
+    },
+
+    {
+      files: ['*.ts'],
+
+      parser: '@typescript-eslint/parser',
+
+      extends: [
+        'airbnb-base',
+        'plugin:import/typescript',
+        'plugin:@typescript-eslint/recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+      ],
+
+      parserOptions: {
+        project: ['./tsconfig.json'],
       },
-    ],
-  },
+
+      plugins: ['@typescript-eslint/eslint-plugin', 'import', 'prettier'],
+
+      settings: {
+        'import/resolver': {
+          node: {
+            extensions: ['.ts'],
+            paths: ['src'],
+          },
+        },
+      },
+      rules: {
+        'import/extensions': [
```

Comment:
This setting is duplicated at line 54:  `'import/extensions': 'off',`

![Screenshot 2022-09-02 at 15 56 47](https://user-images.githubusercontent.com/36277587/188165132-980e1626-d56e-4eed-829f-3c1b3dc2d9ae.png)